### ISSUE TITLE
[audio-slideshow] Use audio-slideshow with notes plugin?

### ISSUE THREAD

**Issue by lechten** [rajgoel/reveal.js-plugins#39]
I'd like to use the `notes` plugin with audio-slideshow. However, when I press `s` to open the speakers view, the audio gets played back several times: In the main browser window and in the new popup window. Besides, if the upcoming slide contains audio, that starts as well. [An example](https://oer.gitlab.io/OS/OS02-Interrupts.html#/slide-kernel-mode) (press `s` to open the speaker's view, then `n` to navigate through the fragments). Is there a workaround to start just one audio?

**Comment by rajgoel** [comment]
As I understand, the audio players are created twice. To solve the problem I think one needs to identify whether the rendering is done for the speaker view and supress the generation of the audio players. Unfortunately, I do not have any experience with the notes plugin.

**Comment by lechten** [comment]
Sorry for leaving this open for so long. Using the following upon initialization as suggested in the closed PR #41 works.

```
condition: function( ) { return !!document.body.classList && !Reveal.isSpeakerNotes(); }
```
